data/data/aws/variables-aws.tf

```diff
@@ -114,56 +114,82 @@ variable "aws_worker_availability_zones" {
   description = "The availability zones to provision for workers.  Worker instances are created by the machine-API operator, but this variable controls their supporting infrastructure (subnets, routing, etc.)."
 }
 
+variable "aws_edge_local_zones" {
+  type    = list(string)
+  default = []
+
+  description = "The zones to provision subnets for the edge pool. Edge instances are created by the machine-API operator, but this variable controls their supporting infrastructure (subnets, routing, etc.)."
+}
+
+variable "aws_edge_parent_zones_index" {
+  type    = map(string)
+  default = {}
+
+  description = <<EOF
+(optional) A map of the Edge's Parent Zone indexes to discover the private's route table ID used by the private subnet from the parent's Zone in the Region.
+Each Local or Wavelength Zone is connected to a parent zone in the Region. If the parent zone has private Route tables, the installer
+uses the index to associate with the private edge subnets.
+
+Example: `{ "us-east-1-nyc-1a"=5, "us-east-1-wl1-nyc-wlz-1"=2 }`
+EOF
+}
+
+variable "aws_edge_zone_groups" {
+  type = list(string)
+  default = []
+  description = "(optional) The list of edge zone groups used to opt-in and create edge subnets."
+}
+
 variable "aws_vpc" {
-  type        = string
-  default     = null
+  type = string
+  default = null
   description = "(optional) An existing network (VPC ID) into which the cluster should be installed."
 }
 
 variable "aws_public_subnets" {
-  type        = list(string)
-  default     = null
+  type = list(string)
+  default = null
   description = "(optional) Existing public subnets into which the cluster should be installed."
 }
 
 variable "aws_private_subnets" {
-  type        = list(string)
-  default     = null
```

Comment:
Im confused why CI was complaining about the recommended space (aligning the '='), I changed this to make CI job passes 🚶🏽‍♂️ 😞

Reply:
@patrickdillon @r4f4 I really don't know what I am doing wrong here as CI TF lint is asking to remove the blank likes. AFAIK the best practice is without this change. Thoughts?

https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_installer/7137/pull-ci-openshift-installer-master-tf-fmt/1668976791244509184

Reply:
```
$ bash hack/tf-fmt.sh 
+ terraform fmt -list -check -diff '-write=false' -recursive data/data/
data/data/aws/variables-aws.tf
--- old/data/data/aws/variables-aws.tf
+++ new/data/data/aws/variables-aws.tf
@@ -171,8 +171,8 @@
 }
 
 variable "aws_internal_zone_role" {
-  type        = string
-  default     = null
+  type = string
+  default = null
   description = "(optional) A role to assume when using an existing hosted zone from another account."
 }
```

Reply:
@mtulio I think it has to do with the terraform binary version. `hack/tf-fmt` (therefore CI) use `quay.io/coreos/terraform-alpine:v0.12.0-rc1`. If you run the format with a newer version (e.g, using the  `v1.3.7` we build with the Installer): 
```diff
$: ./terraform/bin/linux_amd64/terraform fmt -list -check -diff '-write=false' -recursive data/data/aws/variables-aws.tf
data/data/aws/variables-aws.tf
--- old/data/data/aws/variables-aws.tf
+++ new/data/data/aws/variables-aws.tf
@@ -53,12 +53,12 @@
 }

 variable "aws_master_root_volume_type" {
-  type = string
+  type        = string
   description = "The type of volume for the root block device of master nodes."
 }

 variable "aws_master_root_volume_size" {
-  type = string
+  type        = string
   description = "The size of the volume in gigabytes for the root block device of master nodes."
 }
[...]
```

Reply:
> @mtulio The job passed. Did you figure out what was wrong?

I changed[1] the failing line to pass the job, but I don't think it could be the best approach.

It seems we aren't mirroring for a long time; https://quay.io/repository/coreos/terraform-alpine?tab=tags&tag=latest

[1] https://github.com/openshift/installer/compare/3c42b59e4241ecaec78a741ba14d9a6bf8f472f9..449a5ccaabb384395c226e607ae5dcd58089a68f

Reply:
Patrick, as commented on Slack, I opened the following PR to address that improvement isolated; https://github.com/openshift/installer/pull/7339

But I just realized that I can't use the current version of changing the MachinePool attributes as it will generate changes in Install-Config, not proposed initially.

My plan is to revert things, creating a more limited version tomorrow.